### ISSUE TITLE
Consider moving comprehensive summary to the top of validation-testing

### ISSUE THREAD

**Issue by kylebaron** [metrumresearchgroup/mrgvalidate#6]
# Summary

Suggestion: It would be helpful to move that to the top of the doc so it's easy to see the bottom line of the testing

**Comment by seth127** [comment]
Makes sense. Moved it here: https://github.com/metrumresearchgroup/mrgvalidate/commit/7e1ee079d6b8545e13ab6117d5b93f3aa22e6efa

Tested manually and it looks good to me.
